centos/helpers/build.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+set -ex
+echo Installing Centos $CENTOS_RELEASE
+dnf update -y
+dnf install -y \
+    systemd-sysv \
+    vim \
+    binutils \
+    dialog \
+    openssh-server \
+    openssh-clients \
+    sudo \
+    curl \
+    less \
+    man-db \
+    bind-utils \
+    net-tools
+
+
+# disable services we do not need
+systemctl mask systemd-remount-fs.service
+systemctl mask systemd-resolved fstrim.timer fstrim
+systemctl mask e2scrub_reap e2scrub_all e2scrub_all.timer
+
+# systemd does not seem to realize that /dev/null is NOT a terminal
+# under lx but when trying to chown it, it fails and thus the `User=`
+# directive does not work properly ... this little trick fixes the
+# behavior for the user@.service but obviously it has to be fixed in
+# lx :) ...
+touch /etc/systemd/null
+mkdir -p /etc/systemd/system/user@.service.d
+echo "[Service]\nStandardInput=file:/etc/systemd/null\n" \
+> /etc/systemd/system/user@.service.d/override.conf
```

Comment:
I thought we had fixed that in lx quite some time ago ...

Reply:
I think it was fixed in
```
OS-8194 isatty() should return ENOTTY if not a TTY
```